checker/examples/units-extension/Expected.txt

```diff
@@ -0,0 +1,6 @@
+Demo.java:10: error: [assignment.type.incompatible] incompatible types in assignment.
+        frq = 5;
+              ^
+  found   : int
+  required: @Hz (@Hz :: int)
```

Comment:
This is the exact output on command line.

For the shipped units, we have the output format:
Multiples.java:109: error: [assignment.type.incompatible] incompatible types in assignment.
        @m int distance = s * s * accelm;
                                ^
  found   : @mPERs2 int
  required: @m (@org.checkerframework.checker.units.qual.m :: int)

The inner @Hz is simply due to the fact that the unit is not part of any package in the demo.